src/resources/ha-style.html

```diff
@@ -5,53 +5,37 @@
   body {
     font-size: 14px;
 
-    /* for paper-toggle-button */
-    --paper-grey-50: #fafafa;
-    --paper-grey-200: #eeeeee;
-
-    --paper-item-icon-color: #44739e;
-    --paper-item-icon-active-color: #FDD835;
-
+    --primary-color: #03A9F4; /* header */
     --dark-primary-color: #0288D1;
-    --primary-color: #03A9F4;
     --light-primary-color: #B3E5FC;
-    --text-primary-color: #ffffff;
-    --accent-color: #FF9800;
-    --primary-background-color: var(--paper-grey-50);
-    --secondary-background-color: #E5E5E5;
+    --text-primary-color: #FFFFFF; /* header */
+    --primary-background-color: #fafafa;
     --primary-text-color: #212121;
+
+    --secondary-background-color: #E5E5E5; /* states, */
     --secondary-text-color: #727272;
+
+    --tertiary-background-color: #FFFFFF; /* badges, state cards, side menu */
```

Comment:
I don't like it either, I rather have less colors then we have now, but we don't. This is just a way to make it easier to theme, you can still set the individual variables, they are still there. 
But if you don't like it, we can forget it. It won't change functionality.